python/ray/dataframe/dataframe.py

```diff
@@ -1395,6 +1415,22 @@ def iloc(axis=None):
         raise NotImplementedError("Not Yet implemented.")
 
 
+def _get_lengths(df):
+    """Gets the length of the dataframe.
+
+    Args:
+        df: A remote pd.DataFrame object.
+
+    Returns:
+        Returns an integer length of the dataframe object. If the attempt
+          fails, returns 0 as the length.
```

Comment:
`fails` should be indented by another 2 spaces

Reply:
My IDE has failed me.